clarifai/runners/dockerfile_template/Dockerfile.cuda.template

```diff
@@ -7,36 +7,21 @@
 # * Create a non-root user with minimal privileges and use it
 ARG TARGET_PLATFORM=linux/amd64
 ARG CUDA_VERSION=12.4.0
-FROM --platform=$TARGET_PLATFORM nvidia/cuda:${CUDA_VERSION}-base-ubuntu22.04 AS build
-
-ARG DRIVER_VERSION=535
 ARG PYTHON_VERSION=${PYTHON_VERSION}
+FROM --platform=$TARGET_PLATFORM public.ecr.aws/docker/library/python:${PYTHON_VERSION}-slim-bookworm as build
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
     apt-get install --no-install-suggests --no-install-recommends --yes \
-    software-properties-common \
-    gpg-agent && \
-    add-apt-repository ppa:graphics-drivers/ppa && \
-    add-apt-repository ppa:deadsnakes/ppa && \
-    apt-get update && \
-    apt-get install --no-install-suggests --no-install-recommends --yes \
-    python${PYTHON_VERSION} \
-    python${PYTHON_VERSION}-venv \
-    python${PYTHON_VERSION}-dev \
+    software-properties-common  \
     gcc \
-    libpython3-dev \
-    # drivers and nvidia-smi
-    nvidia-utils-${DRIVER_VERSION} \
-    nvidia-driver-${DRIVER_VERSION} \
-    libcap2-bin && \
+    libpython3-dev && \
```

Comment:
@wemoveon2 do we need this you think? I'm worried it'll instal some different version of python from what is in the base image.

Reply:
its the header files needed to build Python extensions. i don't think it will affect installed ver

Reply:
great